VAMobile/src/constants/linking.tsx

```diff
@@ -12,9 +12,40 @@ export const linking: LinkingOptions<any> = {
   config: {
     /* configuration for matching screens with paths */
     screens: {
-      Appointments: 'appointments/:id',
+      Tabs: {
+        screens: {
+          HealthTab: {
+            screens: {
+              ViewMessageScreen: 'messages/:messageID',
+            },
+          },
+        },
+      },
     },
   },
+  // Sets the navigation state for deeply nested screens to ensure navigating backwards works correctly
+  getStateFromPath(path) {
```

Comment:
Will the deep linking tests break if there's a nav change that causes a regression in deep linking? That would be helpful to flag to engineers that they broke something

Reply:
Good point, I just added a test for this so people will know if the back ever navigation breaks